src/main/java/com/google/devtools/build/lib/buildtool/AnalysisPhaseRunner.java

```diff
@@ -225,6 +225,7 @@ private static AnalysisResult runAnalysisPhase(
             request.getCheckForActionConflicts(),
             request.getLoadingPhaseThreadCount(),
             request.getTopLevelArtifactContext(),
+            request.reportIncompatibleTargets(),
```

Comment:
I admittedly don't know anything about the Java style guide, but this seems more readable as `request.incompatibleTargetsShouldBeReported()`. Though obviously more wordy. Otherwise, it makes it seem like this function actually reports targets and returns success/failure of the report maybe? Not sure. Basically phrase it like an assertion.

Anyway, just a stream of consciousness that I had. I guess I should read up on the Java style guide.

EDIT: looking at all the other booleans in this code, looks like this is being consistent. So nevermind!